src/Mod/PartDesign/App/FeatureDraft.cpp

```diff
@@ -216,6 +219,18 @@ App::DocumentObjectExecReturn *Draft::execute()
             neutralPlane = gp_Pln(gp_Pnt(b.x, b.y, b.z), gp_Dir(n.x, n.y, n.z));
         } else if (refPlane->isDerivedFrom<App::Plane>()) {
             neutralPlane = Feature::makePlnFromPlane(refPlane);
+        } else if (refPlane->isDerivedFrom(Base::Type::fromName("Sketcher::SketchObject"))) {
+            Part::Part2DObject* sketch = static_cast<Part::Part2DObject*>(refPlane);

```

Comment:
When it's a cast the class name is already writen. So it is advised to use `auto* sketch` ... to avoid repetition